aiobotocore/httpxsession.py

```diff
@@ -0,0 +1,305 @@
+from __future__ import annotations
+
+import asyncio
+import io
+import os
+import socket
+from typing import TYPE_CHECKING, Any, cast
+
+import botocore
+from botocore.awsrequest import AWSPreparedRequest
+from botocore.httpsession import (
+    MAX_POOL_CONNECTIONS,
+    EndpointConnectionError,
+    HTTPClientError,
+    ReadTimeoutError,
+    create_urllib3_context,
+    ensure_boolean,
+    get_cert_path,
+    logger,
+)
+from multidict import CIMultiDict
+
+import aiobotocore.awsrequest
+from aiobotocore._endpoint_helpers import _text
+
+# TODO: resolve future annotations thing
```

Comment:
what is this about?

Reply:
good question lol
possibly something no longer relevant after you requested minimal typing changes due to following botocore